xml/docu_styleguide.language.xml

```diff
@@ -405,6 +405,49 @@
    </para>
    </sect2>
 
+ <sect2 xml:id="sec-quotation">
+  <title>Quotations</title>
+  <para>
+  Use quotations to quote from sources, such as books.
+  In all other cases, do not use quotation marks:
+  </para>
+  <itemizedlist>
+  <listitem>
+  <para>
+  For computer input, computer output, and user
+  interface elements, use different markup.
+  See also <xref linkend="sec-command-line"/> and
+  <xref linkend="sec-ui-item-markup"/>.
+  </para>
+  </listitem>
+  <listitem>
+  <para>
+  Do not use quotation marks to indicate irony.
+  Avoid irony in technical writing.
+  See also <xref linkend="sec-tone"/>.
+  </para>
+  </listitem>
+  </itemizedlist>
+  <para>
+  To create quotations, use the <tag class="emptytag">quote</tag>
+  element, as it is easier to localize than hardcoded quotation marks and
+  always provides typographic quotes.
+  </para>
+  <para>
+  The period and the comma always go within the quotation marks,
+  as illustrated in <xref linkend="ex-quote"/>. The dash, the semicolon,
+  the colon, the question mark and the exclamation mark go within the
+  quotation marks when they apply to the quoted matter only. They go
+  outside when they apply to the whole sentence.
+  </para>
+  <example xml:id="ex-quote">
+  <title>Quote</title>
+  <para>
+  <quote>Suds may froth,</quote> the sign reads.
+  </para>
```

Comment:
I suppose, example should show the _source code_, not the result, right? As such, you need to use `<screen>` instead of `<para>`. Furthermore you need to mask the `<` characters:

```suggestion
  <screen>&lt;quote>Suds may froth,&lt;/quote> the sign reads.</screen>
```

Reply:
Yeah, I just looked at the tiny example, not the complete guide. Just leave it if the other examples don't do that. Otherwise it would become inconsistent. :slightly_smiling_face: